core/common/src/main/java/alluxio/util/network/NetworkAddressUtils.java

```diff
@@ -582,4 +583,25 @@ public static InetSocketAddress getDataPortSocketAddress(WorkerNetAddress netAdd
       throw Throwables.propagate(e);
     }
   }
+
+  /**
+   * Get master address from zookeeper for the fault tolerant Alluxio masters.
+   *
+   * @param conf the Alluxio Configuration
+   * @return InetSocketAddress
```

Comment:
could you add a bit more detail, like "the master address retrieved from zookeeeper"